src/routes/admin/user.route.js

```diff
@@ -5,5 +5,5 @@ const router = express.Router();
 const userController = require("../../app/controllers/user.controller")
 
 router.delete('/deleteStaff/:id', userController.deleteUser)
-
+router.get('/addstaff', userController.showAddStaffForm.bind(userController))
```

Comment:
ua chỗ này cú pháp lạ z

Reply:
cái này tui copy của TNGB á, sửa lại thành userController.showAddStaffForm cũng được

Reply:
t làm class auth có mấy hàm xài this. mới dùng kiểu này thôi